libPhoneNumber/NBPhoneNumberOfflineGeocoder.h

```diff
@@ -0,0 +1,40 @@
+//
+//  NBPhoneNumberOfflineGeocoder.h
+//  libPhoneNumberiOS
+//
+//  Created by Rastaar Haghi on 6/12/20.
+//  Copyright © 2020 ohtalk.me. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "NBGeocoderMetadataHelper.h"
+#import "NBPhoneNumber.h"
+#import "NBPhoneNumberDefines.h"
+#import "NBPhoneNumberUtil.h"
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NBPhoneNumberOfflineGeocoder : NSObject
+
+- (instancetype)init;
+
+- (NSString *)descriptionForValidNumber:(NBPhoneNumber *)phoneNumber
+                       withLanguageCode:(NSString *)language;
+- (NSString *)descriptionForValidNumber:(NBPhoneNumber *)phoneNumber
+                       withLanguageCode:(NSString *)language
+                         withUserRegion:(NSString *)userRegion;
```

Comment:
User region should be `NSLocale` object

Reply:
We discussed that this wasn't very necessary change.